Use LLVMSymbolizer directly for unwind inside fbcode

zdevito opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #108800

Using LLVMSymbolizer directly avoids having to call fork which has caused timeouts in some circumstances.

Differential Revision: [D49070589](https://our.internmc.facebook.com/intern/diff/D49070589/)

zdevito committed:
Use LLVMSymbolizer directly for unwind inside fbcode

Using LLVMSymbolizer directly avoids having to call fork which has caused timeouts in some circumstances.

Differential Revision: [D49070589](https://our.internmc.facebook.com/intern/diff/D49070589/)

[ghstack-poisoned]

zdevito committed:
Update on "Use LLVMSymbolizer directly for unwind inside fbcode"


Using LLVMSymbolizer directly avoids having to call fork which has caused timeouts in some circumstances.

Differential Revision: [D49070589](https://our.internmc.facebook.com/intern/diff/D49070589/)

[ghstack-poisoned]

zdevito committed:
Update on "Use LLVMSymbolizer directly for unwind inside fbcode"


Using LLVMSymbolizer directly avoids having to call fork which has caused timeouts in some circumstances.

Differential Revision: [D49070589](https://our.internmc.facebook.com/intern/diff/D49070589/)

[ghstack-poisoned]

zdevito committed:
Update on "Use LLVMSymbolizer directly for unwind inside fbcode"


Using LLVMSymbolizer directly avoids having to call fork which has caused timeouts in some circumstances.

Differential Revision: [D49070589](https://our.internmc.facebook.com/intern/diff/D49070589/)

[ghstack-poisoned]

zdevito committed:
Update on "Use LLVMSymbolizer directly for unwind inside fbcode"


Using LLVMSymbolizer directly avoids having to call fork which has caused timeouts in some circumstances.

Differential Revision: [D49070589](https://our.internmc.facebook.com/intern/diff/D49070589/)

[ghstack-poisoned]

zdevito committed:
Update on "Use LLVMSymbolizer directly for unwind inside fbcode"


Using LLVMSymbolizer directly avoids having to call fork which has caused timeouts in some circumstances.

Differential Revision: [D49070589](https://our.internmc.facebook.com/intern/diff/D49070589/)

[ghstack-poisoned]

zdevito committed:
Update on "Use LLVMSymbolizer directly for unwind inside fbcode"


Using LLVMSymbolizer directly avoids having to call fork which has caused timeouts in some circumstances.

Differential Revision: [D49070589](https://our.internmc.facebook.com/intern/diff/D49070589/)

[ghstack-poisoned]

aaronenyeshi reviewed (approved):
LGTM, thank you very much for fixing this issue!

zdevito committed:
Update on "Use LLVMSymbolizer directly for unwind inside fbcode"


Using LLVMSymbolizer directly avoids having to call fork which has caused timeouts in some circumstances.

Differential Revision: [D49070589](https://our.internmc.facebook.com/intern/diff/D49070589/)

[ghstack-poisoned]
